build.gradle

```diff
@@ -90,3 +68,5 @@ tasks.withType(JavaCompile) {
   options.fork = true
   options.incremental = true
 }
+
+project.version = scmVersion.version
```

Comment:
Is this working? I don't see the axion-release plugin anywhere. The plugin allows you to derive the version name from git tags. See here: https://github.com/rakutentech/android-remote-config/blob/master/build.gradle#L36

Reply:
Yup, it displays a nice `0.1.0-SNAPSHOT` at this time, the addition is at `build.gradle:42`.

Reply:
Ah, okay I see. But the it still needs to have the configuration for the plugin so that it uses tags prefixed by `v` i.e. `v1.0.0`.

Reply:
Noted. Thanks.